apps/music/js/metadata/flac.js

```diff
@@ -54,11 +88,9 @@ var FLACMetadata = (function() {
    */
   function findVorbisCommentBlock(blobview) {
     return readMetadataBlockHeader(blobview).then(function(block) {
-      // XXX: Support album art.
-      // See: http://flac.sourceforge.net/format.html#metadata_block_picture
 
       // Did we find a Vorbis comment block yet?
-      if (block.block_type === 4) {
+      if (block.block_type === 4 || block.block_type === 6) {
```

Comment:
We can probably eliminate this line and just have the callback filter out the metadata blocks it doesn't want.